gensrc/proto/types.proto

```diff
@@ -51,6 +51,8 @@ message PTypeNode {
     repeated PStructField struct_fields = 3;
     // only used for complex types, such as array, map and etc.
     optional bool contains_null = 4;
+    // update for map/struct type
+    repeated bool contains_nulls = 5;
```

Comment:
what's the difference to contains_null?

Reply:
contains_nulls is repeated which is vector for null , just keep same with contains null in thrift